### DISCUSSION TITLE
Play store and iOS accept leafmap

### DISCUSSION THREAD

**Discussion by ziskind60** [opengeos/leafmap#361] [Q&A] [answered]
We were hoping to integrate leafmap into our mobile app. Has there ever been a case where the play store and iOS store denied an app because they where using leafmap in the app?

Thanks

**Comment by giswqs** [comment, marked as answer]
I have no experience mobile app development. This is an issue with the upstream packages (e.g., folium, ipyleaflet). If they support mobile, leafmap should be able to support as well.
